docs/internal_api.yml

```diff
@@ -40,12 +40,12 @@ paths:
               $ref: '#/definitions/Error'
   /tokens:
     delete:
-      summary: Delete device tokens. for a tenant with given tenant id
+      summary: Delete device tokens for a tenant with given tenant_id
```

Comment:
Just "Delete device tokens" maybe?
Whole sentence is not exactly true since we added optional device_id parameter.

Reply:
fixed